webpack.config.js

```diff
@@ -65,7 +65,23 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					"sass-loader?data=@import 'assets/stylesheets/shared/component';" +
+						'&outputStyle=' + ( bundleEnv === 'production' ? 'compressed' : 'nested' ),
+				]
+			},
+			{
+				test: /\/client\/components\/vertical-nav.+index\.jsx$/,
+				loaders: [
+					'imports?style=./style.scss,withStyles=isomorphic-style-loader/lib/withStyles',
+					'exports?default=withStyles.default( style )( exports.default )'
```

Comment:
Another problem is, that in a single folder there might be many `.jsx?` components, but only the "entry point" component should have the `import`.